cockroachcloud/quickstart.md

```diff
@@ -112,70 +79,61 @@ For this tutorial, we will use the [`movr` workload](../stable/movr.html) to run
     ~~~
     </section>
 
-3. Initialize the `movr` workload using the `cockroach workload` command and the [connection string](#step-4-get-the-connection-string).
-
-    In the connection string, the SQL user's username is prepopulated. Replace `<password>` with the SQL user's password that you entered in [Step 2](#step-2-create-a-sql-user). Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 4](#step-4-get-the-connection-string).
+1. In your terminal, connect to your cluster using the [connection string](#step-2-set-up-your-cluster-connection):
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ cockroach workload init movr \
-    'postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    $ cockroach sql --url 'postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster_name>.defaultdb?sslmode=verify-full&sslrootcert=<certs_dir>/cc-ca.crt'
     ~~~
 
-4. Use the built-in SQL client to view the database:
+    In the connection string copied from the **Connection info** modal, your username, password and cluster name are pre-populated. Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 2](#step-2-set-up-your-cluster-connection). For example:
 
     {% include copy-clipboard.html %}
     ~~~ shell
-    $ cockroach sql \
-    --url='postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    $ cockroach sql --url 'postgres://maxroach:password123@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/test-cluster.defaultdb?sslmode=verify-full&sslrootcert=Users/maxroach/certs/cc-ca.crt'
     ~~~
 
+1. Using built-in SQL client, you can now run [CockroachDB SQL statements](learn-cockroachdb-sql.html):
```

Comment:
dropped article "the built-in..."  - also is there a link to the `cockroach sql` doc in cloud world?

Reply:
er, can we link to the `cockroach sql` doc *from* cloud world? (I meant)